pages/reader/epub.vue

```diff
@@ -91,27 +91,46 @@
                 />
               </template>
               <template v-if="isShowTextToSpeechOptions">
-                <UButton
-                  icon="i-material-symbols-skip-previous-rounded"
-                  variant="ghost"
-                  :disabled="!isTextToSpeechOn"
-                  @click="skipBackward"
-                />
-                <USelect
-                  v-model="ttsLanguageVoice"
-                  :items="ttsLanguageVoiceOptions"
-                />
-                <USelect
-                  v-model="ttsPlaybackRate"
-                  icon="i-material-symbols-speed-rounded"
-                  :items="ttsPlaybackRateOptions"
-                />
-                <UButton
-                  icon="i-material-symbols-skip-next-rounded"
-                  variant="ghost"
-                  :disabled="!isTextToSpeechOn"
-                  @click="skipForward"
-                />
+                <USlideover
+                  :title="$t('reader_tts_options_button')"
```

Comment:
Would be nice if we can make the title go away, such that the slide over would just cover the height of original header, then user can still read the book content below when using TTS control
Not sure if possible in `<USlideover>` though
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/fffeba20-6604-4a35-9d08-a9209abdef79" />